static/css/style.css

```diff
@@ -36,10 +36,29 @@ a.disabled {
     width: 210px;
 }
 
-.rightBorder{
-    border: 5px solid;
-}
-
 td a div{
     padding-top: 20px;
 }
+
+.borderLeft {
+    border-left: 5px;
+}
```

Comment:
There's border classes in bootstrap if you want them, instead of creating new classes [link](https://getbootstrap.com/docs/5.0/utilities/borders/)